src/registry/event-catalog.js

```diff
@@ -35,6 +38,20 @@ class EventCatalog {
 		this.events = [];
 
 		this.EndpointFactory = EventEndpoint;
+
+		this.on("broker.event", ctx => {
```

Comment:
Why we should you event emitter to call a non-await-ed method? I feel it too expensive. Not enough just skip the `await` in the `callEventHandler`